docs/plan/payment-flow-improvement-proposal.md

```
@@ -0,0 +1,378 @@
+
+1. Remove `saveGtmPaymentEventInLocalStorage` calls
+2. Remove `getGtmPaymentEventFromLocalStorage` calls
+3. Remove `removeGtmPaymentEventFromLocalStorage` calls
+4. Deprecate/remove `gtmPaymentEventLocalStorage.ts`
+
+## Implementation Plan
+
+### BE Changes (packages/framework + packages/frontend-api)
+
+**If Option B (computed BE field):**
+| Task | File | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)